AI21_Summarize_example_model_use.ipynb

```diff
@@ -104,38 +104,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "! pip install -U \"ai21[AWS]>=2.0.3\"\n",
+    "! pip install -U \"ai21[AWS]>=2.1.2\"\n",
     "import ai21"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Check the version of ai21 - must be 2.0.3 or higher"
+    "### Check the version of ai21 - must be 2.1.2 or higher"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
Is that an empty cell? Perhaps could be deleted?

Reply:
It's a cell with code where I removed an output that existed